torch/onnx/_internal/fx/onnxfunction_dispatcher.py

```diff
@@ -235,45 +270,64 @@ def get_aten_name(
     def get_function_overloads(
         self,
         node: torch.fx.Node,
-        aten_name: str,
+        domain: str,
+        op_name: str,
+        overload: Optional[str],
         diagnostic_context: diagnostics.DiagnosticContext,
-    ) -> Set[Union["onnxscript.OnnxFunction", "onnxscript.TracedOnnxFunction"]]:
+    ) -> Tuple[
+        Set[registration.SymbolicFunction], Optional[Set[registration.SymbolicFunction]]
+    ]:
         """Get the function overloads from the registry.
 
         Args:
             node: The node to get the function overloads for.
-            aten_name: The aten name of the node.
+            domain: The domain of the node.
+            op_name: The name of the node.
+            overload: The FX overload of the node. It's None if the node is default overload.
             diagnostic_context: The diagnostic context to use for reporting errors.
 
         Returns:
-            Set of function overloads.
+            Set of SymbolicFunctions.
         """
-        function_group = None
 
-        if self.onnx_registry.is_registered_op(aten_name, self.opset_version):
-            function_group = self.onnx_registry.get_function_group(aten_name)
+        # NOTE: If the ATen/Custom operators are nt registered, these two groups will be None.
+        # And non-registerd ATen/Custom operators will trigger error in the next step.
+        function_group: Optional[Set[registration.SymbolicFunction]] = None
+        custom_function_group: Optional[Set[registration.SymbolicFunction]] = None
 
-        # Fall back to overloadpacket name: eg: aten.add.Tensor -> aten::add
+        if self.onnx_registry.is_registered_op(
+            domain=domain, op_name=op_name, overload=overload
+        ):
+            function_group = self.onnx_registry.get_functions(
+                domain=domain, op_name=op_name, overload=overload
+            )
+            custom_function_group = self.onnx_registry._get_custom_functions(
+                domain=domain, op_name=op_name, overload=overload
+            )
+
+        # NOTE: Fall back to overloadpacket name: eg: aten.add.Tensor -> aten::add
+        # Support default fallback for overloadpacket.
         elif hasattr(
             node.target, "overloadpacket"
```

Comment:
2. only keep node and drop the names does seem to be doable? we need processing in get_aten_name to get the name from node. A more complicated way would be that we move get_aten_name into OpName to support from_node().

1. drop node. I will need your help on this one. It's doable but I think this can be in another PR.

Reply:
Moved `_get_aten_name` into `get_function_overloads` to delete naming input for `get_function_overloads`.